mhkit/utils.py

```diff
@@ -76,6 +86,43 @@ def get_statistics(data,freq,period=600):
 
     return means,maxs,mins,stdevs
 
+def vector_averaging(data):
+    """
+    Function used to calculate statistics for vector/directional channels
+
+    Parameters
+    ----------
+    data : pandas Series, numpy array, list
+        Vector channel to calculate statistics on
+    
+    Returns
+    -------
+    vector_avg : numpy array
+        Vector mean statistic
+    vector_std : numpy array
+        Vector standard deviation statistic
+    """
+    try: data = np.array(data)
+    except: pass
+    assert isinstance(data, np.ndarray), 'data must be of type np.ndarray'
+    
+    # calculate mean
+    Ux = sum(np.sin(data*np.pi/180))/len(data)
+    Uy = sum(np.cos(data*np.pi/180))/len(data)
+    vector_avg = (90 - np.arctan2(Uy,Ux)*180/np.pi)
+    if vector_avg<0: vector_avg = vector_avg+360
+    elif vector_avg>360: vector_avg = vector_avg-360
+    # calculate standard deviation              
+    magsum = round((Ux**2 + Uy**2)*1e8)/1e8 # round to 8th decimal place to reduce roundoff error
+    epsilon = (1-magsum)**0.5
+    if not np.isreal(epsilon): # check if epsilon is imaginary (error)
+        vector_std = 0
+        print('WARNING: vector averaging error in calculating epsilon')
+    else:
+        vector_std = np.arcsin(epsilon)*(1+0.1547*epsilon**3)*180/np.pi
+
+    return vector_avg, vector_std
```

Comment:
I had the same thought....i initially left it as `vector_averaging` since its what everyone refers to although i agree its technically not appropriate. I have updated this to be called `vector_statistics` instead. I think thats close enough to where people should be able to recognize what its for.